### ISSUE TITLE
Remove dummy alternate descriptions

### ISSUE THREAD

**Issue by nkutsche** [data2type/xmlgraphics-fop#2]
In case it is executed in PDF/UA mode, FOP generates dummy alternate descriptions like "No alternate text specified" for links (and figures?). They are completely useless and just avoids failures of the Accessibility checker.

The text shouldn't be generated any more. FOP should throw an error if a required alternate description is not provided by `fox:alt`.

**Comment by nkutsche** [comment]
Two remarks:

* This is annoying as the alternate description is also missplaced for links (see #5)
* This could be implemented depending on a (new) a11y-policy: like `strict` (generate real a11y PDFs) or `lax` (just avoid a11y check failures)

**Comment by nkutsche** [comment]
Documentation about events that need to be thrown:

https://xmlgraphics.apache.org/fop/trunk/events.html

**Comment by nkutsche** [comment]
Configuration build entrypoind: `org.apache.fop.configuration.DefaultConfigurationBuilder` in fop-core
